packages/editor/src/client/routes/commentRatingEditView.tsx

```diff
@@ -0,0 +1,250 @@
+import {ApolloError} from '@apollo/client'
+import PlusIcon from '@rsuite/icons/legacy/Plus'
+import SpinnerIcon from '@rsuite/icons/legacy/Spinner'
+import TrashIcon from '@rsuite/icons/legacy/Trash'
+import React, {useCallback, useEffect, useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {
+  Button,
+  FlexboxGrid,
+  Form,
+  IconButton,
+  Loader,
+  Message,
+  Modal,
+  SelectPicker,
+  toaster
+} from 'rsuite'
+
+import {
+  CommentRatingSystemAnswer,
+  FullCommentRatingSystem,
+  RatingSystemType,
+  useCreateRatingSystemAnswerMutation,
+  useDeleteRatingSystemAnswerMutation,
+  useRatingSystemLazyQuery,
+  useUpdateRatingSystemMutation
+} from '../api/index'
+import {IconButtonTooltip} from '../atoms/iconButtonTooltip'
+import {createCheckedPermissionComponent} from '../atoms/permissionControl'
+
+const showErrors = (error: ApolloError): void => {
```

Comment:
Have you produced an error? I think it would not work, React would throw an invalid hook call: https://reactjs.org/warnings/invalid-hook-call-warning.html

Reply:
Works fine and throws no warning as it's called inside a hook down there.
What I forgot though was adding it to delete and update